### ISSUE TITLE
console.log() logs incomplete objects

### ISSUE THREAD

**Issue by patsalv** [remotion-dev/remotion#2473]
```tsx
	console.log('aaaaa', {
		hi: 'there',
		p: 'a',
		x: 'd',
		w: 'x',
		l: 'd',
		k: 'd',
		a: 'q',
		m: 'o',
		abc: '/public/subtitles.srt',
	});

```

in verbose mode logs: 

 src/Composition.tsx:90  aaaaa { hi: "there", p: "a", x: "d", w: "x", l: "d" }

**Comment by JonnyBurger** [comment]
Code: https://github.com/remotion-dev/remotion/blob/7c77fbaf7306eaac900d1133811a10a905d8bce8/packages/renderer/src/format-logs.ts#L25


Test: https://github.com/remotion-dev/remotion/blob/7c77fbaf7306eaac900d1133811a10a905d8bce8/packages/renderer/src/test/format-logs.test.ts#L14

**Comment by JonnyBurger** [comment]
Fixed by #2498
